12-target-details.tex

```diff
@@ -28,6 +28,134 @@ \subsection{Loading extern classes using "require" function}
 \section{Flash}
 \label{target-flash}
 
+\subsection{Getting started with Flash}
+\label{target-flash-getting-started}
+
+Developing Flash applications is really easy with Haxe. Let's see our first sample.
+This is a very simple example showing most of the toolchain. 
+
+Create a new folder, save this class as \ic{Main.hx}.
+
+\begin{lstlisting}
+import flash.Lib;
+import flash.display.Shape;
+class Main {
+    static function main() {
+        var stage = Lib.current.stage;
+        
+        // create a center aligned rounded gray square
+        var shape = new Shape();
+        shape.graphics.beginFill(0x333333);
+		shape.graphics.drawRoundRect(0, 0, 100, 100, 10);
+		shape.x = (stage.stageWidth - 100) / 2;
+		shape.y = (stage.stageHeight - 100) / 2;
+		
+		stage.addChild(shape);
+    }    
+}
+\end{lstlisting}
+
+To compile, run from the command line:
+
+\begin{lstlisting}
+haxe -swf main-flash.swf -main Main -swf-version 15 -swf-header 960:640:60:f68712
+\end{lstlisting}
+
+.. or create and run (double-click) a file called \ic{compile.hxml}. In this example the hxml-file should be in the same directory as the example class.
+
+\begin{lstlisting}
+-swf main-flash.swf
+-main Main
+-swf-version 15
+-swf-header 960:640:60:f68712
+\end{lstlisting}
+
+The output will be a main-flash.swf with size 960x640 pixels at 60 FPS with a orange background color and a gray square in the center.
+
+\paragraph{Display the Flash}
+
+Run the SWF standalone using the \href{https://www.adobe.com/support/flashplayer/downloads.html}{Standalone Debugger FlashPlayer}. 
+
+To display the output in a browser using the Flash-plugin, create a html-document called \ic{index.html} and open it.
```

Comment:
Because the H is pronounced as if it starts with an A in this case and it depends on the pronunciation. The same often applies to a leading X which sounds like EX.